helloworld.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+class HelloWorld:
+    def hello(self):

```

Comment:
Add another argument called 'num' that will be used to control how many times "Hello World!" is printed